src/link/patch.c

```diff
@@ -320,6 +320,42 @@ static int32_t computeRPNExpr(struct Patch const *patch,
 			}
 			break;
 
+		case RPN_SIZEOF_SECT:
+			name = (char const *)expression;
```

Comment:
This is unsafe, as we're not guaranteed to have a NUL byte in the rest of the expression's buffer. This would cause the `getRPNByte` below to report an *internal* error, which happens to be fatal and stop the program. If this is intentional, please add a comment.

Note also that it relies on `getRPNByte` not e.g. `free`ing the expression somehow. I think it'd be best to document that somewhere.

Reply:
I copied that approach from the `RPN_BANK_SECT` case above; will comment all three of them.